pkg/kvm/nvme.go

```diff
@@ -68,7 +75,7 @@ func (s *Server) CreateNvmeController(ctx context.Context, in *pb.CreateNvmeCont
 
 	// Create request can miss Name field which is generated in spdk bridge.
 	// Use subsystem instead, since it is required to exist
-	dirName := filepath.Base(in.NvmeController.Spec.SubsystemNameRef)
+	dirName := filepath.Base(subsys)
```

Comment:
in listener numbered controller id was used as dir name. Here, a string subsystem id, provided at creation is used. Let's align and use for example `nqn`(it should be reachable in all cases)